src/config.ts

```diff
@@ -2,6 +2,15 @@ import dotenv from "dotenv";
 
 dotenv.config();
 
-export const SERVER_PORT = process.env.SERVER_PORT
-  ? Number(process.env.SERVER_PORT)
-  : 3001;
+// AWS SDK
```

Comment:
With the GC Forms web app I feel like configuration variables are spread across the whole codebase and are not always easy to locate.

What I would like to propose for this project is to have all configuration variables defined in this `config.ts` file.